src/main/java/seedu/address/logic/commands/FilterCommand.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.logic.commands;
+
+/**
+ * Filters the displayed interview or person list based on the date or status provided.
+ */
+public abstract class FilterCommand extends Command {
+
+    public static final String COMMAND_WORD = "filter";
```

Comment:
Is this String constant necessary if the subclasses have their own respective `final` command words as well?